cranelift/codegen/src/machinst/blockorder.rs

```diff
@@ -106,6 +106,8 @@ pub struct BlockLoweringOrder {
     /// which is used by VCode emission to sink the blocks at the last
     /// moment (when we actually emit bytes into the MachBuffer).
     cold_blocks: FxHashSet<BlockIndex>,
+    /// These are loop headers. Used for alignment.
+    loop_headers: FxHashSet<BlockIndex>,
```

Comment:
Would it make sense to merge `cold_blocks` and `loop_headers` into a single `SecondaryMap` with a bitflag as element? Or are both still too infrequently used such that it causes a large increase in memory usage?